docs/development.md

```diff
@@ -4,40 +4,55 @@ This guide explains how to set up your environment for developing on Helm and Ti
 
 ## Prerequisites
 * Docker 1.10 or later
-* Docker Compose 1.6 or later
-* A Kubernetes cluster with Helm/Tiller installed (optional)
+* A Kubernetes cluster with Helm/Tiller installed
+* Telepresence 0.75 or later
 * kubectl 1.2 or later (optional)
 * Git
 
 ## Running Monocular
 
-We use [Docker Compose](https://docs.docker.com/compose/) to orchestrate the UI frontend and API backend containers for local development. The simplest way to get started is:
+We develop Monocular in a Kubernetes environment. Minikube can be used to run a
```

Comment:
Maybe you can give some context on why k8s is needed? For example mentioning that the chart import job uses native k8s cronjobs